kss/builder/decanter/kss-assets/css/kss.css

```diff
@@ -1684,7 +1684,11 @@ body {
     -ms-grid-row: 1;
     -ms-grid-column: 1; }
     .masthead .brand-logo a {
-      color: #ffffff; }
+      color: #ffffff;
+      text-decoration: none;
+      font-weight: 700; }
+      .masthead .brand-logo a:hover, .masthead .brand-logo a:focus {
```

Comment:
Do we want a hover/focus color for the Decanter logo? This might be a @kerri-augenstein question?

Reply:
I did put a hover / focus color on it. I used `$color-cloud`, which I had found in the alert variants that use white text on a dark(ish) background. Not a great difference, IMO, but the precedent was there.

Reply:
Yeah, the color is ok, but since we don't have hover state for the lockup or sitename (Redwood), perhaps we just don't do color change for these logo type things for consistency?